Assets/MixedRealityToolkit/_Core/Devices/OpenVR/GenericOpenVRController.cs

```diff
@@ -124,44 +149,27 @@ public void UpdateController(XRNodeState xrNodeState)
         {
             UpdateControllerData(xrNodeState);
 
-            Debug.Assert(Interactions != null, "No interaction configuration for controller");
+            Debug.Assert(Interactions != null, "No interactions configuration for controller!");
+
             if (Interactions == null) { Enabled = false; }
 
             for (int i = 0; i < Interactions?.Length; i++)
             {
-                switch (Interactions[i].InputType)
+                switch (Interactions[i].AxisType)
                 {
-                    case DeviceInputType.None:
+                    case AxisType.None:
                         break;
-                    case DeviceInputType.SpatialPointer:
-                        UpdatePointerData(Interactions[i]);
+                    case AxisType.Digital:
```

Comment:
With this change, [the line in `UpdateButtonData`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/8e6554e86d8627c13251544af9463de6d0486d50/Assets/MixedRealityToolkit/_Core/Devices/OpenVR/GenericOpenVRController.cs#L358) that allows for converting a `SingleAxis` to an `OnInputDown/Up` event will never be called.

Reply:
I'll take a look. Trying to get controller registration solid before going in depth on input.

Reply:
Fixed.